.github/workflows/build.yaml

```diff
@@ -0,0 +1,61 @@
+name: Build
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      V: 1
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        include:
+          - os: ubuntu-latest
+            mingw: true
+            linux: true
+          - os: macos-latest
+            darwin: true
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Set up MinGW
+        if: ${{ matrix.mingw }}
+        uses: egor-tensin/setup-mingw@v2
+        with:
+          platform: x64
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Install lerna
+        run: npm install -g lerna
+
+      - name: Build project
+        run: lerna run build
```

Comment:
Doing `npm install -g lerna` means that it will always use the latest version of lerna to build, instead of what's defined in the lockfile. We should remove that step, and run `npx lerna run build` instead of trying to run `lerna` directly.